sf-pcap.c

```diff
@@ -834,9 +848,41 @@ pcap_dump_open(pcap_t *p, const char *fname)
 	return (pcap_setup_dump(p, linktype, f, fname));
 }
 
+#ifdef _WIN32
+/*
+ * Initialize so that sf_write() will output to a stream wrapping the given raw
+ * OS file HANDLE.
+ */
+pcap_dumper_t *
+pcap_dump_hopen(pcap_t *p, intptr_t osfd)
+{
+	int fd;
+	FILE *file;
+
+	fd = _open_osfhandle(osfd, _O_APPEND);
+	if (fd < 0) {
+		pcap_fmt_errmsg_for_errno(p->errbuf, PCAP_ERRBUF_SIZE,
+		    errno, "_open_osfhandle");
+		return NULL;
+	}
+
+	file = _fdopen(fd, "wb");
+	if (file == NULL) {
```

Comment:
Shouldn't there be a `_close(fd);` here?

Reply:
Yes, there should.

And there should be one in `pcap_hopen_offline_with_tstamp_precision()` as well; I fixed that one in 40fe5f21bae1847a92f18efdcc8ad061db8f6cce.

Reply:
Well spotted! Fixed here as well in 71cb091